python/lsst/ts/standardscripts/auxtel/atdome/slew_dome.py

```diff
@@ -64,8 +64,6 @@ def get_schema(cls):
                 description: Azimuth position (in degrees) to slew the dome to.
                 anyOf:
```

Comment:
Thanks for adopting this recommendation. Since it only allows the one type, no need to do the anyOf here.